tools/report-converter/codechecker_report_converter/plist_converter.py

```diff
@@ -98,8 +98,9 @@ def _get_analyzer_type(self):
     def _create_diag(self, message, files):
         """ Creates a new plist diagnostic from the given message. """
         fmap = {files[i]: i for i in range(0, len(files))}
+        checker_name = message.checker if message.checker else self.tool_name
         diag = {'location': self._create_location(message, fmap),
-                'check_name': message.checker,
+                'check_name': checker_name,
```

Comment:
Minor detail, but there is no need to introduce an extra variable which is used only once, the expression from above could be inserted here. Question of style, you may choose according to your preference.

Reply:
I used a separate variable to avoid breaking long line:
```py
diag = {'location': self._create_location(message, fmap),
        'check_name': message.checker if message.checker else \
            self.tool_name
```

```py
checker_name = message.checker if message.checker else self.tool_name
diag = {'location': self._create_location(message, fmap),
        'check_name': checker_name
```
For me the second format is much more readable.

Reply:
Alright, I didn't notice that the line was too long.